base/compiler/ssair/show.jl

```diff
@@ -693,10 +716,11 @@ end
 _strip_color(s::String) = replace(s, r"\e\[\d+m" => "")
 
 # corresponds to `verbose_linetable=true`
-function ircode_verbose_linfo_printer(code::IRCode, used::BitSet)
+function ircode_verbose_linfo_printer(code::IRCode)
```

Comment:
@vtjnash, @Keno: I am tempted to just remove the verbose lineinfo printer for IRCode alltogether, since it's now possible to use the same printer as we already use for `CodeInfo`, which shows the same information. Thoughts?

Reply:
That makes sense to me. Are there any situations where it is preferred? In principle, `debuginfo` should be able to take other keywords, to indicate how precisely to show the information. The compact printer (i.e. the one previously used for IRCode) could be called `oneliner` or `inline` to make it accessible to `code_warntype`.

Reply:
> Are there any situations where it is preferred?

It doesn't mess with alignment like the default printer for `source` does and is a bit more subtle without the yellow, but other than appearance, they shouldn't be any different.

> The compact printer (i.e. the one previously used for IRCode) could be called oneliner or inline to make it accessible to code_warntype.

Right now this one only works for IRCode. It probably wouldn't be too hard to adapt for CodeInfo as well, but I haven't yet found a strong need for it, since for unoptimized code there naturally isn't that much debuginfo to show. I'd leave that part out for now, but I can change the name so we don't have to change the name again.

Reply:
I will leave removing `ircode_verbose_linfo_printer` for a followup PR, so it doesn't obscure the purpose of this PR and in case anyone is interested in it again it's easier to retrieve. I did rename `ircode_defaul_linfo_printer` to `inline_linfo_printer`.